docs/en/stack/security/authentication/token-authentication-services.asciidoc

```diff
@@ -0,0 +1,57 @@
+[role="xpack"]
+[[token-authentication-services]]
+=== Token-based authentication services
+
+The {stack-security-features} authenticate users by using realms and one or more token-based
+authentication services followed by the <<realms,realms>>. The
```

Comment:
> followed by the <<realms,realms>>.

Is this necessary? Is it meant to indicate that we look at the services or the realms in a specific order? If so, I think this should be clarified in a separate sentence.